Tests/IntegrationTests/Configuration.cs

```diff
@@ -1,86 +1,83 @@
-﻿using System.Configuration;
+﻿using System;
+using System.Configuration;
 using System.Security.Cryptography.X509Certificates;
 using Digst.OioIdws.OioWsTrust;
+using Digst.OioIdws.Rest.Client;
 
 namespace DK.Gov.Oio.Idws.IntegrationTests
 {
     public class Configuration
```

Comment:
Der er et eller andet her - der vel være to forskellige konfigurationer alt efter om det er .NET eller Java, for både SOAP og REST WSP, samt REST WSC.

Reply:
Selvfølgelig. Det holder ikke helt det jeg fik lavet der.